### PR TITLE
Create sys admin via cli

### PR DESCRIPTION
**Pull request by shmanu017**
Added steps to create SysAdmin via CLI

### CODE REVIEW

File: enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md [hunk 1]
```diff
@@ -221,4 +221,30 @@ To verify a user was successfully granted the SysAdmin role, ensure they can do
 
 - Navigate to `grafana.BASEDOMAIN`
 - Navigate to `kibana.BASEDOMAIN`
-- Access the 'System Admin' tab from the top left menu of the Astronomer UI
\ No newline at end of file
+- Access the 'System Admin' tab from the top left menu of the Astronomer UI
+
+#### Create SysAdmin User via CLI
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md:226]: @shmanu017 Can you bump up this section to above "Verify SysAdmin Access"? I'd say these headers can go:

Assign Users System-Level Roles > Create SysAdmin User via Astro CLI > Verify SysAdmin Access

File: enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md [hunk 2]
```diff
@@ -221,4 +221,30 @@ To verify a user was successfully granted the SysAdmin role, ensure they can do
...
+
+Get name of one of Houston pods(switch to base namespace: astronomer (default))
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md:228]: How would a user get a Houston pod? If it's switching to the namespace, add the command in here maybe?

File: enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md [hunk 3]
```diff
@@ -221,4 +221,30 @@ To verify a user was successfully granted the SysAdmin role, ensure they can do
...
+
+- Get a Prisma security token via `kubectl -n astronomer exec astronomer-houston-****** -ti -- npm run token` (filling in that name in place of Houston in this case)
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md:230]: I'd here say:

> Now, get a Prisma security token by running:
> 
> ```
> $ kubectl -n astronomer exec astronomer-houston-****** -ti -- npm run token
> ```
> 
> Make sure to substitute your full Houston pod name above.

File: enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md [hunk 4]
```diff
@@ -221,4 +221,30 @@ To verify a user was successfully granted the SysAdmin role, ensure they can do
...
+
+
+- Run `kubectl -n astronomer port-forward svc/astronomer-prisma 4466:4466`
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md:233]: > Then, run:
> 
> ```
> $ kubectl -n astronomer port-forward svc/astronomer-prisma 4466:4466
> ```

File: enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md [hunk 5]
```diff
@@ -221,4 +221,30 @@ To verify a user was successfully granted the SysAdmin role, ensure they can do
...
+
+
+- Visit http://localhost:4466/houston
+
+
+- Click on `HTTP Headers` in the bottom left and set `{"Authorization": "<token from 1st step>"}`
```
Comment (paolaperaza) [inline enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md:239]: Is this the prisma security token? I'd specify! So,

```
{"Authorization": "<prisma-security-token>"}
```

(Or replace with another token)

File: enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md (hunk 1 above)
Comment (paolaperaza) [reply enterprise/v0.16/06_manage-astronomer/02_manage-platform-users.md:226]: Alright, reading through this again - this isn't through the CLI, then right? This is more through Houston? Ignorant q, but when would users prefer to add a SysAdmin via this method?

I think if you included one more sentence with context of what this method is, that'd be helpful!

**Review by paolaperaza — CHANGES_REQUESTED** [review]
Thanks for doing this @shmanu017 ! Added a few comments - happy to take a look again later today or whenever you have time to reply back.

**Comment by paolaperaza** [comment]
Ah, @shmanu017 I just saw the second file you added on the SSL Cert. Can you actually take that commit out of this PR and open up a new one for that doc, just so we can review/talk about it separately? Sorry I didn't notice earlier!

cc: @pritt20

**Comment by shmanu017** [comment]
@paolaperaza I have made all the necessary changes as requested, Please review the document.

**Comment by shmanu017** [comment]
https://www.notion.so/astronomerio/Create-SysAdmin-via-cli-b463bf83242a4735b8bfa29a7b32ccf1

Making an internal Q&A note

**Pull request closed without merging**